src/reader/plugins/mark_as_read.py

```diff
@@ -75,6 +76,7 @@ def _get_config(reader, feed_url, key, patterns_key):
 
 
 _CONFIG_TAG = 'mark-as-read'
+_MARK_AS_READ_BY_TAG_SUFFIX = 'once'
```

Comment:
Nit: `_ONCE_TAG_SUFFIX` may be a better name for this constant – `ONCE` to show what it's for, and without `MARK_AS_READ` because it doesn't add much information (this whole module is about that).

Reply:
Modified as suggested.